lib/Maintenance/Tasks/RedirectCleanupTask.php

```diff
@@ -17,7 +17,7 @@
 namespace Pimcore\Maintenance\Tasks;
 
 use Pimcore\Maintenance\TaskInterface;
-use Pimcore\Model\Redirect;
+use Pimcore\Bundle\SeoBundle\Model\Redirect;
```

Comment:
We should move this task to the bundle too since there should not be a dependency in the core and the maintenance. No Cleanup is needed if the bundle is not enabled or installed

Reply:
We still need to move this file to the bundle.